torch/csrc/inductor/aoti_torch/shim_mkldnn.cpp

```diff
@@ -523,3 +523,19 @@ AOTITorchError aoti_torch_cpu__mkl_linear(
 #endif // AT_MKL_ENABLED
 
 #endif // AT_MKLDNN_ENABLED()
+
+AOTITorchError aoti_torch_cpu__weight_int4pack_mm_cpu_tensor(
```

Comment:
But are we using mkldnn here?

Reply:
No. It does not require MKLDNN. We are considering renaming these files from mkldnn to cpu. Thanks.

Reply:
@jgong5 , currently, the `shim_mkldnn` indicates that it should only serve oneDNN. However, Weiwen and I checked the motivation as it is CPU-dedicated and cannot be reused across different hardware backends. For XPU, we have provided `shim_xpu` regardless of its implementation being on the top of oneDNN. That's why we want to rename the file to CPU. Does it make sense to you?